pkg/kubelet/network/dns/dns.go

```diff
@@ -123,7 +123,7 @@ func (c *Configurer) formDNSSearchFitsLimits(composedSearch []string, pod *v1.Po
 	if limitsExceeded {
 		log := fmt.Sprintf("Search Line limits were exceeded, some search paths have been omitted, the applied search line is: %s", strings.Join(composedSearch, " "))
 		c.recorder.Event(pod, v1.EventTypeWarning, "DNSConfigForming", log)
-		klog.ErrorS(nil, "eventlog", log)
+		klog.ErrorS(nil, log)
```

Comment:
IMHO it's still weird to pass nil as the error here.
Personally I would create an error instead of a string two lines earlier, using `fmt.Errorf`, then modify the `ErrorS` call to supply a message like "limits exceeded".

Reply:
Updated